src/frontend/README.md

```diff
@@ -44,14 +40,24 @@ else.
 
 ### Metrics
 
-- Automatic Instrumentation (Agent): :large_blue_diamond:
-- Automatic Instrumentation (OTel Operator): :red_circle:
-- Instrumentation Libraries  : :red_circle:
-- Manual Metric Creation:  :red_circle:
-- Collector Metric Transformation::red_circle:
-- Push Metrics: :red_circle:
-- SLO Metrics: :red_circle:
-- Multiple Metric Instruments: :red_circle:
+- [Automatic Instrumentation (OTel
+  Operator)](https://github.com/open-telemetry/opentelemetry-operator):
+  :red_circle:
+- [Instrumentation
+  Libraries](https://opentelemetry.io/docs/concepts/instrumenting-library/):
+  :red_circle:
+- [Manual Metric
+  Creation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#manual-instrumentation)):
+  :red_circle:
+- [Collector Agent Metric
+  Transformation](https://opentelemetry.io/docs/collector/deployment/#agent)::red_circle:
+- [Push
+  Metrics](https://opentelemetry.io/docs/reference/specification/metrics/sdk/#push-metric-exporter):
+  :red_circle:
+- [SLO Metrics](https://github.com/openslo/openslo#slo): :red_circle:
+- [Multiple Metric
+  Instruments](https://opentelemetry.io/docs/reference/specification/metrics/api/#synchronous-and-asynchronous-instruments):
```

Comment:
Is this an expansion of Ln49 (Manual Metric Creation)? If yes, consider rewording it to "Manual instrumentation using multiple instruments"

Reply:
Updated to Multiple Manual Metric Instruments